client/app.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const App = () => {
+    return <div>
```

Comment:
@AStashevskaya Looks like this code is not formatted with your `prettier` settings